packages/ui/src/TextField.tsx

```diff
@@ -174,6 +174,7 @@ export const TextField: FC<TextFieldProps> = ({
           }}
           accessibilityHint="Enter text here"
           accessibilityLabel="Text input field"
+          accessibilityState={{disabled}}
```

Comment:
I need to do a little more research on this I couldn't get my screen reader to read that the text was actually disabled when I tested it out on the demo site. It certainly doesn't hurt to have this - possibly one only works on certain screenreaders/devices. We can also update the accessibility label to share this info if needed.